packages/fx-core/resource/package.nls.json

```diff
@@ -501,8 +501,8 @@
   "core.selectValidateMethodQuestion.validate.schemaOptionDescription": "Validate using manifest schema",
   "core.selectValidateMethodQuestion.validate.appPackageOption": "Validate app package using validation rules",
   "core.selectValidateMethodQuestion.validate.appPackageOptionDescription": "Validate app package using validation rules",
-  "core.selectValidateMethodQuestion.validate.testCasesOption": "Publish Readiness",
-  "core.selectValidateMethodQuestion.validate.testCasesOptionDescription": "Check your app with Microsoft's test cases before publishing",
+  "core.selectValidateMethodQuestion.validate.testCasesOption": "Validate against pre-publish test cases",
```

Comment:
Validate all required test cases before publishing.
Logic: To simplify it so that users would understand and perceive it correctly.

Reply:
The tests are not required tests. I would prefer "Validate all integration test cases before pubishing"